src/test/components/ThreadActivityGraph.test.js

```diff
@@ -189,5 +192,26 @@ describe('SelectedThreadActivityGraph', function() {
       );
       expect(getCallNodePath()).toEqual(['A', 'B', 'C', 'F', 'G']);
     });
+
+    it('does nothing if a filtered out sample is clicked', function() {
+      const { dispatch, stackGraphCanvas, getCallNodePath } = setup();
+      // Get a selection by clicking a visible stack.
+      fireEvent(
+        stackGraphCanvas,
+        getMouseEvent('mouseup', { pageX: getSamplesPixelPosition(1) })
+      );
+      expect(getCallNodePath()).toEqual(['A', 'B', 'C', 'F', 'G']);
+
+      // Search for something that will cause a stack to hide.
+      dispatch(changeCallTreeSearchString('C'));
+
+      // Click on the position of the hidden stack, and confirm that
+      // the selection did not change.
+      fireEvent(
+        stackGraphCanvas,
+        getMouseEvent('mouseup', { pageX: getSamplesPixelPosition(2) })
```

Comment:
Can you please add a comment stating what `2` computes to? As I understand this would be `A -> B -> H  -> I`.